Providers/Scripts/2.4x-2.5x/Scripts/nxOMSAutomationWorker.py

```diff
@@ -0,0 +1,40 @@
+#!/usr/bin/env python
+# ====================================
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# See license.txt for license information.
+# ====================================
+import os
+import sys
+
+
+def init_locals(Name):
+    if Name is None:
+        Name = ''
+    return Name.encode('ascii', 'ignore')
+
+
+def Set_Marshall(Name):
+    Name = init_locals(Name)
+    return [0]
+
+
+def Test_Marshall(Name):
+    Name = init_locals(Name)
+    return [0]
+
+
+def Get_Marshall(Name):
+    arg_names = list(locals().keys())
+    Name = init_locals(Name)
+    retval = 0
+    retd = {}
+    ld = locals()
+    for k in arg_names:
+        retd[k] = ld[k]
+    return retval, retd
```

Comment:
@D1v38om83r Just for my information - You'd be adding the implementation in a separate check-in? If yes, then why not get the PR done after the implementation is done? As once you push this change to master, it'll start generating the nxAutomationWorker.zip in the daily builds (without the python implementation)